Cargo.toml

```diff
@@ -37,39 +37,39 @@ prover-utils = { path = "crates/prover-utils" }
 agglayer-telemetry = { git = "https://github.com/agglayer/agglayer.git", branch = "release/0.2.1" }
 
 # Core dependencies
-alloy = { version = '0.11.0', features = ["full"] }
-alloy-primitives = { version = "0.8.15", features = ["serde", "k256"] }
+alloy = { version = '0.11.1', features = ["full"] }
+alloy-primitives = { version = "0.8.21", features = ["serde", "k256"] }
 anyhow = "1.0.94"
 arc-swap = "1.7.1"
-async-trait = "0.1.82"
+async-trait = "0.1.86"
 base64 = "0.22.0"
 bincode = "1.3.3"
 buildstructor = "0.5.4"
-clap = { version = "4.5.23", features = ["derive", "env"] }
+clap = { version = "4.5.30", features = ["derive", "env"] }
 dirs = "5.0.1"
 dotenvy = "0.15.7"
 ethers = "2.0.14"
 ethers-gcp-kms-signer = "0.1.5"
 ethers-signers = "2.0.14"
 futures = "0.3.31"
 hex = "0.4.3"
-hyper = "1.5.2"
+hyper = "1.6.0"
 jsonrpsee = { version = "0.24.7", features = ["full"] }
 lazy_static = "1.5.0"
 parking_lot = "0.12.3"
-prost = "0.13.4"
+prost = "0.13.5"
 reqwest = "0.12"
 serde = { version = "1.0.216", features = ["derive"] }
-serde_json = "1.0.133"
+serde_json = "1.0.138"
 serde_with = "3.11.0"
 thiserror = "2.0.7"
 tokio = { version = "1.42.0", features = ["full"] }
 tokio-stream = { version = "0.1.17", features = ["sync"] }
 tokio-util = "0.7.13"
-toml = "0.8.15"
+toml = "0.8.20"
 tonic = { version = "0.12.3", default-features = false }
 tonic-types = "0.12.3"
-tower = "0.4.13"
+tower = "0.5.2"
```

Comment:
@Freyskeyd Are you aware of any significant changes in the `tower` major version that would require some changes on our side?

Reply:
Beside many things put behind features, some types and traits have been changed, so not a trivial update.